internal/freelist/hashmap.go

```diff
@@ -74,17 +105,21 @@ func (f *freelist) hashmapAllocate(txid common.Txid, n int) common.Pgid {
 	return 0
 }
 
-// hashmapReadIDs reads pgids as input an initial the freelist(hashmap version)
-func (f *freelist) hashmapReadIDs(pgids []common.Pgid) {
-	f.init(pgids)
+func (f *HashMap) Count() int {
+	return f.FreeCount() + f.PendingCount()
+}
```

Comment:
It has identical implementation as array.Count(), so let's move it into shared?

Reply:
I wish, `FreeCount()` is implementation dependent, so we can't move it into shared. We can deal with it the same way mergeSpans works however

Reply:
added it to the shared interface